students/AlyssaHong/lesson03/list_lab.py

```diff
@@ -32,11 +39,12 @@
 
 # Display all the fruits that begin with “P”
 for x in fruits:
-    if x.startswith('P'):
+    if x.startswith('P') or x.startswith('p'):
```

Comment:
you could also do `if fruit.upper().startswith('P')` 

best to use good variable names instead of `x` -> `for fruit in fruits`